src/main/webapp/app/shared/util/global.utils.ts

```diff
@@ -93,7 +93,8 @@ export const onError = (alertService: AlertService, error: HttpErrorResponse) =>
             alertService.error('error.http.405');
             break;
         case 500:
-            alertService.error('error.http.500');
+            // Removed to avoid alerts about internal errors as the user can't do anything about it
+            // and the alert does not provide any other value
```

Comment:
Question: What happens now when there is an internal server error? Will the user not be notified at all?
The user might not be able to do anything about the error, but he can still go ahead and report them here on GitHub. If there is no alert at all, this would no longer happen.

Reply:
Yes, nothing would show up. In the long run, proper error handling has to be done by affected components to show meaningful error messages. They are still captured by Sentry, though.
This was a wish of @krusche